pkg/agent/containerd/containerd.go

```diff
@@ -93,7 +93,7 @@ func Run(ctx context.Context, cfg *config.Node) error {
 
 		addDeathSig(cmd)
 		if err := cmd.Run(); err != nil {
-			fmt.Fprintf(os.Stderr, "containerd: %s\n", err)
+			logrus.Errorf("containerd failed: %s", err)
```

Comment:
nit: in other places we say "exited" when things exit unexpectedly
```suggestion
			logrus.Errorf("containerd exited: %s", err)
```

Reply:
thanks! Done